app/logins.py

```diff
@@ -133,18 +133,15 @@ def get_login_kinds():
     return [
         {
             "method": "github",
-            "button": "data:image/png;base64," + GITHUB_IMAGE_B64,
-            "text": "Log in with GitHub",
+            "name": "GitHub",
         },
         {
             "method": "gitlab",
-            "button": "data:image/svg+xml;base64," + GITLAB_IMAGE_B64,
-            "text": "Log in with Gitlab",
+            "name": "Gitlab",
```

Comment:
```suggestion
            "name": "GitLab",
```

According to usage on https://about.gitlab.com/